docs/how-to-install.md

```diff
@@ -6,145 +6,170 @@ license: "This software is licensed under the Apache License version 2."
+2. Extract the Gravitino package and you should have these directories and files:
 
-#### Configure the Apache Gravitino server
+   ```text
+   |── ...
+   └── distribution/package
+      |── bin/
+      |   ├── gravitino.sh                        # Gravitino server Launching scripts.
+      |   └── gravitino-iceberg-rest-server.sh    # Gravitino Iceberg REST server Launching scripts.
+      |── catalogs
+      |   └── hadoop/                             # Apache Hadoop catalog dependencies and configurations.
+      |   └── hive/                               # Apache Hive catalog dependencies and configurations.
+      |   └── jdbc-doris/                         # JDBC doris catalog dependencies and configurations.
+      |   └── jdbc-mysql/                         # JDBC MySQL catalog dependencies and configurations.
+      |   └── jdbc-postgresql/                    # JDBC PostgreSQL catalog dependencies and configurations.
+      |   └── kafka/                              # Apache Kafka PostgreSQL catalog dependencies and configurations.
+      |   └── lakehouse-iceberg/                  # Apache Iceberg catalog dependencies and configurations.
+      |   └── lakehouse-paimon/                   # Apache Paimon catalog dependencies and configurations.
+      |── conf/                                   # All configurations for Gravitino.
+      |   ├── gravitino.conf                      # Gravitino server and Gravitino Iceberg REST server configuration.
+      |   ├── gravitino-iceberg-rest-server.conf  # Gravitino server configuration.
+      |   ├── gravitino-env.sh                    # Environment variables, etc., JAVA_HOME, GRAVITINO_HOME, and more.
+      |   └── log4j2.properties                   # log4j configuration for the Gravitino server and Gravitino Iceberg REST server.
+      |── libs/                                   # Gravitino server dependencies libraries.
+      |── logs/                                   # Gravitino server and Gravitino Iceberg REST server logs. Automatically created after the server starts.
+      |── data/                                   # Default directory for the Gravitino server to store data.
+      |── iceberg-rest-server/                    # Gravitino Iceberg REST server package and dependencies libraries.
+      └── scripts/                                # Extra scripts for Gravitino.
+   ```
 
-The Gravitino server configuration file is `conf/gravitino.conf`. You can configure the Gravitino server by modifying this file. Basic configurations have already been added to this file. All the configurations are listed in [Gravitino Server Configurations](./gravitino-server-config.md).
+### Initialize the RDBMS (Optional)
 
-#### Configure the Apache Gravitino server log
+If you want to use the `relational` backend storage, you need to initialize the RDBMS.
```

Comment:
With the removal of the K-V store, I think this is no longer an optional step and would be required but not 100% sure.

Reply:
let's leave it here for now to see if someone can confirm it.